CHANGES.rst

```diff
@@ -13,6 +13,10 @@ Deprecated
 Features added
 --------------
 
+* #12727: build: Allow :option:`--pdb` to debug warnings when
```

Comment:
I think you need `sphinx --option` and not just `--option` when referencing them (on my phone so hard to check the CI sometimes).

Reply:
I'm not sure how to make this xref work but I don't think it really matters too much.

Reply:
I tried `sphinx-build --pdb` but that didn't work.

Reply:
Mmh let's forget about it. It should have worked though but maybe the options are not documented? Anyway, my suggestion was to help users that read the changelog so that they know the effect of each option (I can't really help since I'm on a phone now)